data/ExperimentDesignPresets.ts

```diff
@@ -0,0 +1,37 @@
+import {ExperimentDesign} from "../types/experiments";
+
+// Utility type to being able to specify partials of nested objects
+type RecursivePartial<T> = {
+  [P in keyof T]?:
+    T[P] extends Array<infer U> ? Array<RecursivePartial<U>> :
+    T[P] extends {[key: string]: any} ? RecursivePartial<T[P]> :
+    T[P];
+};
+
+// Utility type for specifiying a partial of any experiment design
+interface Preset<T> {
+  name: string;
+  description: string;
+  preset: RecursivePartial<T>
+}
+
+const presets :  {[id: string] : Preset<ExperimentDesign> } = {
+  empty_aa: {
+    name: "A/A Experiment",
+    description: "A design for diagnostic testing of targeting or enrollment. Fixed to 1% of the population.",
+    preset: {
+      branches: [
+        {slug: "control", ratio: 1, value: null},
+        {slug: "treatment", ratio: 1, value: null}
+      ],
+      bucketConfig: {
+        randomizationUnit: "normandy_id",
+        start: 0,
+        count: 100,
```

Comment:
I think neither of these have any meaning here.  `start` will be determined by the bucket allocator, and `count` I think will be determined by @tdsmith 's forthcoming audience/feature population size logic

Reply:
:+1:

Reply:
makes sense i think this is old logic. will remove